### DISCUSSION TITLE
Blog post example of GET request with consumer key and secret

### DISCUSSION THREAD

**Discussion by trosel** [lexmag/oauther#26] [Show and tell]
http://www.eidel.io/2017/07/13/http-api-client-elixir-noun-project

Just sharing this because it might help someone else.

**Comment by GusGA** [comment]
Thanks @WildYorkies, that blog post was really useful to me today.

**Comment by trosel** [comment]
@GusGA Thanks, it made my day that something I posted randomly was useful! For some reason, HTTP seems more complex in Elixir than other languages. Not sure if it's the syntax or libraries or what :S
